_data/areas.yml

```
@@ -1,17 +1,33 @@
-- title: Radiation Transport in Complex Geometries
+- code: rtnam
+  title: Radiation Transport and Nuclear Analysis Methods
   url: https://svalinn.github.io
   image: hab1.png
   alt-text: Cut-away of NASA habitat module showing radiation deposition
-  desc: Open source software tools that support Monte Carlo radiation transport on complex CAD-based geometries, with workflows that 
+  desc: "Open source software tools that support Monte Carlo radiation transport on complex CAD-based geometries, 
+        with workflows that extend the scope of analysis"
 
-- title: Nuclear Fuel Cycle Analysis
+- code: nsa
+  title: Nuclear Systems Analysis
+  url: https://svalinn.github.io
+  image: parastell.png
+  alt-text: A sample stellarator geometry showing blanket layers and magnets
+  desc: Analysis of specific nuclear systems in support of design and licensing
+
+- code: nfca
+  title: Nuclear Fuel Cycle Analysis
   url: https://fuelcycle.org
   image: cyclus.png
   alt-text: Cyclus logo - a gear with a letter C inscribed
   desc: Open source software tools that allow flexible modeling of nuclear fuel cycles over varying time and physical fidelities.
```

None of the others have a period at the end
```suggestion
  desc: Open source software tools that allow flexible modeling of nuclear fuel cycles over varying time and physical fidelities
```